glean-core/android/src/main/java/mozilla/telemetry/glean/scheduler/PingUploadWorker.kt

```diff
@@ -118,7 +117,7 @@ class PingUploadWorker(context: Context, params: WorkerParameters) : Worker(cont
                     // Process the upload response
                     LibGleanFFI.INSTANCE.glean_process_ping_upload_response(incomingTask, result)
                 }
-                PingUploadTask.Wait -> SystemClock.sleep(THROTTLED_BACKOFF_MS)
+                is PingUploadTask.Wait -> SystemClock.sleep(action.time)
```

Comment:
Why is it we use `is <item>` here but not on the line below? Can we make that the same on all match arms for consistency?

Reply:
Ugh, nice catch, this is a leftover from development. I has a bug and thought that was the fix, but it wasn't. I'll just remove it.